lib/avrora/encoder.ex

```diff
@@ -51,6 +51,33 @@ defmodule Avrora.Encoder do
     codec.decode(payload)
   end
 
+  @doc """
+  Decode binary Avro message with :plain format and given schema.
+
+  This is pulled out as a special function to work around :plain encoded binaries that,
+  due to their encoded data, start with the magic byte.
+
+  ## Examples
+
+      ...> payload = <<0, 232, 220, 144, 233, 11, 200, 1>>
+      ...> Avrora.Encoder.decode_plain(payload,"io.confluent.numeric_transfer")
+      {:ok, %{ "link_is_enabled" => false, "updated_at" => 1_586_632_500, "updated_by_id" => 1_00 }
+  """
+  @spec decode_plain(binary(), schema_name: String.t()) ::
+          {:ok, map() | list(map())} | {:error, term()}
```

Comment:
Can you please inline it with line 66? I have a limit of 120 chars per line and will be good to utilize it

Reply:
The formatting with line_length 120 seems to cause some issues (see [CI run of suggested changes](https://github.com/Strech/avrora/pull/70/commits/240c9688372a90c476da99211a2077cc189e7445)) I reverted some of them back to what the formatter seems to be happy with. 

If I may suggest: To enforce the formatter to have line length 120, you'd probably want to change the .formatter.exs file and add a `line_length: 120` argument?

Reply:
Hmmm, sorry, my bad mixing 2 different projects. All good then 👍🏼